### ISSUE TITLE
WARN - ains.python.pydev.PyStackFrame - Timeout waiting for response

### ISSUE THREAD

**Issue by gnuhub** [gnuhub/sync-engine#14]

**Comment by gnuhub** [comment]
```
packaging tool: /Volumes/data/anaconda/envs/py27/bin/python /Applications/PyCharm.app/Contents/helpers/packaging_tool.py list 
2018-06-14 12:03:56,150 [ 261028]   WARN - ains.python.pydev.PyStackFrame - Timeout waiting for response on 114 
com.jetbrains.python.debugger.PyDebuggerException: Timeout waiting for response on 114
	at com.jetbrains.python.debugger.pydev.AbstractCommand.execute(AbstractCommand.java:140)
	at com.jetbrains.python.debugger.pydev.RemoteDebugger.loadFrame(RemoteDebugger.java:142)
	at com.jetbrains.python.debugger.pydev.MultiProcessDebugger.loadFrame(MultiProcessDebugger.java:176)
	at com.jetbrains.python.debugger.PyDebugProcess.loadFrame(PyDebugProcess.java:721)
	at com.jetbrains.python.debugger.PyStackFrame.a(PyStackFrame.java:135)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:305)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-06-14 12:03:56,151 [ 261029]   WARN - ains.python.pydev.PyStackFrame - Process is running 
com.jetbrains.python.debugger.PyDebuggerException: Process is running
	at com.jetbrains.python.debugger.PyDebugProcess.f(PyDebugProcess.java:846)
	at com.jetbrains.python.debugger.PyDebugProcess.loadFrame(PyDebugProcess.java:717)
	at com.jetbrains.python.debugger.PyStackFrame.a(PyStackFrame.java:135)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:305)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)

```

**Comment by gnuhub** [comment]
```
WARN - ains.python.pydev.PyStackFrame - No connection (command:  114 ) 
com.jetbrains.python.debugger.PyDebuggerException: No connection (command:  114 )
	at com.jetbrains.python.debugger.pydev.AbstractCommand.execute(AbstractCommand.java:138)
	at com.jetbrains.python.debugger.pydev.RemoteDebugger.loadFrame(RemoteDebugger.java:142)
	at com.jetbrains.python.debugger.pydev.MultiProcessDebugger.loadFrame(MultiProcessDebugger.java:176)
	at com.jetbrains.python.debugger.PyDebugProcess.loadFrame(PyDebugProcess.java:721)
	at com.jetbrains.python.debugger.PyStackFrame.a(PyStackFrame.java:135)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:305)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-06-14 12:02:08,415 [ 153293]   WARN - ains.python.pydev.PyStackFrame - Disconnected 
com.jetbrains.python.debugger.PyDebuggerException: Disconnected
	at com.jetbrains.python.debugger.PyDebugProcess.f(PyDebugProcess.java:836)
	at com.jetbrains.python.debugger.PyDebugProcess.loadFrame(PyDebugProcess.java:717)
	at com.jetbrains.python.debugger.PyStackFrame.a(PyStackFrame.java:135)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:305)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)

```

**Comment by gnuhub** [comment]
https://intellij-support.jetbrains.com/hc/en-us/community/posts/205967904-pydev-debugger-is-not-connecting

**Comment by gnuhub** [comment]
```
source activate py27
python /Applications/PyCharm.app/Contents/helpers/pydev/setup_cython.py build_ext --inplace
```

**Comment by gnuhub** [comment]
```
2018-06-14 12:31:41,528 [ 273140]   INFO - python.packaging.PyPackageUtil - Refreshing installed packages for SDK /Volumes/data/anaconda/envs/py27/bin/python 
2018-06-14 12:31:41,528 [ 273140]   INFO - packaging.PyPackageManagerImpl - Running packaging tool: /Volumes/data/anaconda/envs/py27/bin/python /Applications/PyCharm.app/Contents/helpers/packaging_tool.py list 
2018-06-14 12:31:43,166 [ 274778]   INFO - packaging.PyPackageManagerImpl - Running packaging tool: /Volumes/data/anaconda/envs/py27/bin/python /Applications/PyCharm.app/Contents/helpers/packaging_tool.py list 
2018-06-14 12:32:09,390 [ 301002]   INFO -  #git4idea.commands.GitHandler - [sync-engine] git -c core.quotepath=false -c log.showSignature=false rm --ignore-unmatch --cached -- .idea/inspectionProfiles/profiles_settings.xml 
2018-06-14 12:32:28,698 [ 320310]   WARN - ckaging.InstalledPackagesPanel - Cannot fetch the latest version of the installed package contextlib2 
java.net.SocketTimeoutException: connect timed out
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at sun.security.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:673)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:175)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
	at sun.net.www.protocol.https.HttpsClient.<init>(HttpsClient.java:264)
	at sun.net.www.protocol.https.HttpsClient.New(HttpsClient.java:367)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:191)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1181)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1032)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:177)
	at sun.net.www.protocol.http.HttpURLConnection.followRedirect0(HttpURLConnection.java:2701)
	at sun.net.www.protocol.http.HttpURLConnection.followRedirect(HttpURLConnection.java:2623)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1806)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1474)
	at java.net.HttpURLConnection.getResponseCode(HttpURLConnection.java:480)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getResponseCode(HttpsURLConnectionImpl.java:338)
	at com.intellij.util.io.HttpRequests.openConnection(HttpRequests.java:498)
	at com.intellij.util.io.HttpRequests.access$300(HttpRequests.java:49)
	at com.intellij.util.io.HttpRequests$RequestImpl.getConnection(HttpRequests.java:278)
	at com.intellij.util.io.HttpRequests$RequestImpl.getInputStream(HttpRequests.java:287)
	at com.intellij.util.io.HttpRequests$RequestImpl.getReader(HttpRequests.java:305)
	at com.intellij.util.io.HttpRequests$RequestImpl.getReader(HttpRequests.java:298)
	at com.jetbrains.python.packaging.PyPIPackageUtil$2.a(PyPIPackageUtil.java:99)
	at com.intellij.util.io.HttpRequests.lambda$doProcess$0(HttpRequests.java:403)
	at com.intellij.util.net.ssl.CertificateManager.runWithUntrustedCertificateStrategy(CertificateManager.java:335)
	at com.intellij.util.io.HttpRequests.doProcess(HttpRequests.java:403)
	at com.intellij.util.io.HttpRequests.process(HttpRequests.java:383)
	at com.intellij.util.io.HttpRequests.access$100(HttpRequests.java:49)
	at com.intellij.util.io.HttpRequests$RequestBuilderImpl.connect(HttpRequests.java:252)
	at com.jetbrains.python.packaging.PyPIPackageUtil$2.load(PyPIPackageUtil.java:99)
	at com.jetbrains.python.packaging.PyPIPackageUtil$2.load(PyPIPackageUtil.java:93)
	at com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3716)
	at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2424)
	at com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2298)
	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2211)
	at com.google.common.cache.LocalCache.get(LocalCache.java:4154)
	at com.google.common.cache.LocalCache.getOrLoad(LocalCache.java:4158)
	at com.google.common.cache.LocalCache$LocalLoadingCache.get(LocalCache.java:5147)
	at com.jetbrains.python.packaging.PyPIPackageUtil.a(PyPIPackageUtil.java:272)
	at com.jetbrains.python.packaging.PyPIPackageUtil.a(PyPIPackageUtil.java:220)
	at com.jetbrains.python.packaging.PyPIPackageUtil.c(PyPIPackageUtil.java:247)
	at com.jetbrains.python.packaging.PyPIPackageUtil.d(PyPIPackageUtil.java:256)
	at com.jetbrains.python.packaging.PyPIPackageUtil.fetchLatestPackageVersion(PyPIPackageUtil.java:291)
	at com.jetbrains.python.packaging.ui.PyPackageManagementService.a(PyPackageManagementService.java:383)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at com.intellij.util.concurrency.BoundedTaskExecutor$2.run(BoundedTaskExecutor.java:212)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-06-14 12:32:39,323 [ 330935]   INFO -              #AgreementUpdater - Error updating privacy policy: java.security.cert.CertificateException: java.security.SignatureException: Signature length not correct: got 128 but was expecting 512 
```

**Comment by gnuhub** [comment]
Cannot fetch the latest version of the installed package contextlib2

**Comment by gnuhub** [comment]
Cannot fetch the latest version of the installed package gevent-socketio

**Comment by gnuhub** [comment]
fix by #17
